src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -11,48 +11,67 @@
 
 namespace Symfony\Component\Filesystem\Tests;
 
+use Symfony\Component\Filesystem\Tests\Fixtures\StringishObject;
+
 /**
  * Test class for Filesystem.
  */
 class FilesystemTest extends FilesystemTestCase
 {
-    public function testCopyCreatesNewFile()
+    public function provideSourceTargetPaths()
     {
-        $sourceFilePath = $this->workspace.DIRECTORY_SEPARATOR.'copy_source_file';
-        $targetFilePath = $this->workspace.DIRECTORY_SEPARATOR.'copy_target_file';
+        $workspace = $this->createWorkspace();
```

Comment:
@stof i spotted and fixed that yesterday... https://github.com/symfony/symfony/pull/19132/commits/ab42b5f7580f43cd967794c033b22e9b7aa9d72a

Is it incorrect?